client/js/router/index.js

```diff
@@ -8,6 +8,9 @@ import Rules from '../views/Rules.vue';
 import Stats from '../views/Stats.vue';
 import store from '../store/store.js';
 
+export const ROUTE_NAME_LOBBY = 'Lobby';
+export const ROUTE_NAME_GAME = 'Game';
```

Comment:
Need these as consts to keep the references DRY as we need to use these values both here, and inside the auth status check action.